docs/intro/how-to-bridge-lava.md

```diff
@@ -0,0 +1,239 @@
+
+## 1.2 **Add Arbitrum Network in MetaMask**
+1. Open MetaMask and click the network selector (usually labeled "Ethereum Mainnet").
+2. Click "Add Network", then add/approve Arbitrum if it's not already in your list.
+3. You can verify the Arbitrum network details at (arbitrum-mainnet.infura.io).
+
+   ![MetaMask settings for adding or switching to the Arbitrum network pt1](/img/tutorial/how_to_bridge/MetaMask1.png)
+
+   ![MetaMask settings for adding or switching to the Arbitrum network pt2](/img/tutorial/how_to_bridge/MetaMask_Arbitrum.png)
+   
+   ![MetaMask settings for adding or switching to the Arbitrum network pt3](/img/tutorial/how_to_bridge/MetaMask_Arbitrum2.png)
+
+---
+
+## 1.3 **Import the Lava (Arbitrum) Token Contract**
+1. Copy the official Lava on Arbitrum **ERC20 contract address** from a reliable source:
+- **Arbiscan:** https://arbiscan.io/token/0x11e969e9B3f89cB16D686a03Cd8508C9fC0361AF
+- **Copy & Paste:** `0x11e969e9B3f89cB16D686a03Cd8508C9fC0361AF`
+- **Lava Website:** https://www.lavanet.xyz/blog/eruption 
+
+2. In MetaMask, click **Import Tokens**, paste the contract address, and confirm.
+
+    ![MetaMask interface with the Lava (Arbitrum) token contract being manually imported. pt1](/img/tutorial/how_to_bridge/MetaMask_LavaImport.png)
+
+    ![MetaMask interface with the Lava (Arbitrum) token contract being manually imported. pt2](/img/tutorial/how_to_bridge/MetaMask_LavaImport2.png)
+
+- Once imported, you should see your Lava (Arbitrum) tokens in your MetaMask wallet.
+
+    Updated MetaMask token list including Lava (Arbitrum) tokens.](/img/tutorial/how_to_bridge/MetaMask_ArbImport.png)
+
+<br/>
+
+## 1.4 **Set Up a Lava Mainnet Wallet**
+
+Because Lava Mainnet is a Cosmos chain, you need a **Cosmos-compatible wallet:**
+
+1. Install the **Leap** or **Keplr** Chrome extension.
+- Leap: https://app.leapwallet.io/
+- Keplr: https://wallet.keplr.app/
+2. Create or import your wallet, and keep your seed phrase secure.
+
+- **Leap wallet Chrome extension interface** 
+
+    ![Leap wallet opened with “Lava” selected as the chain, showing the user’s wallet address. pt1](/img/tutorial/how_to_bridge/Leap_1.png)
+
+    ![Leap wallet opened with “Lava” selected as the chain, showing the user’s wallet address. pt2](/img/tutorial/how_to_bridge/Leap_2.png)
+
+- **Keplr wallet Chrome extension interface**
+
+    ![Keplr wallet showing the Lava chain selection and the wallet address. pt1](/img/tutorial/how_to_bridge/Keplr_1.png)
+
+    ![Keplr wallet showing the Lava chain selection and the wallet address. pt2](/img/tutorial/how_to_bridge/Keplr_2.png)
+
+<br/>
+
+## 2. **Using Skip:Go to Bridge**
+
+1. Open Skip:Go [URL: https://go.skip.build/] and connect your MetaMask wallet
+- **Skip:Go homepage**
+
+    o’s main dashboard, showing an option to connect MetaMasks.](/img/tutorial/how_to_bridge/SkipGo_2.png)
```

Comment:
I think here we have something missing at the beginning because the image is not rendering. I'm guessing it should be `![Skip:Go's main dashboard...`